Symfony/CS/Tests/AbstractIntegrationTest.php

```diff
@@ -180,7 +180,8 @@ protected function doTestIntegration(IntegrationCase $case)
             throw new IOException(sprintf('Failed to write to tmp. file "%s".', $tmpFile));
         }
 
-        $changed = $fixer->fixFile(new \SplFileInfo($tmpFile), $case->getFixers(), false, true, new FileCacheManager(false, null, $case->getFixers()));
+        $fixers = $case->getFixers();
+        $changed = $fixer->fixFile(new \SplFileInfo($tmpFile), $fixers, false, true, new FileCacheManager(false, null, $fixers));
```

Comment:
In my opinion it should be another pr, together with the change in `IntegrationCase`.